app/src/util/helpers.ts

```diff
@@ -1,3 +1,25 @@
+export const getTranslationFromDictionary = (
+  translations: Record<string, string> | string | undefined,
+  lang?: string,
```

Comment:
Usually abbreviated as `lng` in the codebase currently, but lang works too I suppose...